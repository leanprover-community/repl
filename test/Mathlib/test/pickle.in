{"cmd": "import Mathlib\nopen BigOperators\nopen Real\nopen Nat"}

{"cmd": "theorem mathd_algebra_455\n  (x : Nat)\n  (h : 2 * (2 * (2 * (2 * x))) = 48) :\n  x = 3 := by sorry", "env": 0}

{"pickleTo": "test/pickle.olean", "proofState": 0}

