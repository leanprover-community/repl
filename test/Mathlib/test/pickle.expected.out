{"env": 0}

{"sorries":
 [{"proofState": 0,
   "pos": {"line": 4, "column": 14},
   "goal": "x : ℕ\nh : 2 * (2 * (2 * (2 * x))) = 48\n⊢ x = 3",
   "endPos": {"line": 4, "column": 19}}],
 "messages":
 [{"severity": "warning",
   "pos": {"line": 1, "column": 8},
   "endPos": {"line": 1, "column": 25},
   "data": "declaration uses 'sorry'"}],
 "env": 1}

{"proofState": 1, "goals": ["x : ℕ\nh : 2 * (2 * (2 * (2 * x))) = 48\n⊢ x = 3"]}

